apps/music/index.html

```diff
@@ -18,7 +18,8 @@
   <link rel="stylesheet" href="/css/app.css">
   <!-- Deferred Scripts -->
   <script defer src="/components/bridge/bridge.js"></script>
-  <script defer src="/shared/js/intl/l20n.js" type="application/javascript;version=1.8"></script>
+  <script defer src="/shared/js/intl/l20n-service.js" type="application/javascript;version=1.8"></script>
+  <script defer src="/shared/js/intl/l20n-client.js" type="application/javascript;version=1.8"></script>
```

Comment:
Is there any common code between these? At least in the case of bridge.js, there is shared code between client.js and service.js which is why they put a full bridge.js in there for cases where you need both to reduce the overhead. Just wondering.